openedx/core/djangoapps/site_configuration/helpers.py

```diff
@@ -2,19 +2,14 @@
 Helpers methods for site configuration.
 """
 
-
 from django.conf import settings
+from edx_django_utils.cache import RequestCache
 
 
-def get_current_site_configuration():
+def _uncached_get_current_site_configuration():
```

Comment:
Can we use the [request_cached](https://github.com/edx/edx-platform/blob/master/openedx/core/lib/cache_utils.py#L22) decorator here?

Reply:
Beautiful, I'll check that out!

Reply:
OK great, that certainly simplified things. :-)